### PR TITLE
Add New Port Squeek the Meek

### PR DESCRIPTION
**Pull request by MadShmupper**
Add New Port Squeek the Meek

# New Port for { Squeek the Meek }

## Game Information
- **Title**: { Squeek the Meek }
- **URL**: { https://foletto.itch.io/squeek-the-meek }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/squeek/port.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "version": 3,
+  "name": "squeek.zip",
+  "items": [
+    "Squeek the Meek.sh",
+    "squeek/"
```
Comment (JanTrueno) [inline ports/squeek/port.json:6]: Why is there a "/" after "squeek"?

**Review by JanTrueno — CHANGES_REQUESTED** [review]

File: ports/squeek/port.json (hunk 1 above)
Comment (MadShmupper) [reply ports/squeek/port.json:6]: Been found it this way some time now. It shows it filled out in the example

Comment (JanTrueno) [reply ports/squeek/port.json:6]: You're right, it does show that in the example. Never seen it :)

**Review by JanTrueno — APPROVED** [review]

File: ports/squeek/port.json (hunk 1 above)
Comment (MadShmupper) [reply ports/squeek/port.json:6]: Ahh..ancient gamemaker secret😉😂❤

**Pull request merged**